### ISSUE TITLE
the 

### ISSUE THREAD

**Issue by liudonglei** [castorini/Castor#143]
so far, 2018-08-18. 
the data path using in the Castor/sm_cnn/create_dataset.sh such as ''../../Castor-data/TrecQA''
is NOT match with the real path in Castor-data dir.

can you please check it?

**Comment by Victor0118** [comment]
@liudonglei You are right. Current SMCNN code needs refactoring. See https://github.com/castorini/Castor/issues/128. Welcome to create your PR to contribute!

**Comment by liudonglei** [comment]
@Victor0118   Dataset path mismatch 的问题可以手动地将Castor-data/datasets下的trecqa和wikiqa目录拷贝到上层目录解决。
但是我陷入了这个问题，详见 https://github.com/castorini/Castor/issues/142
由于我对torchtext包一窍不通，琢磨不出这个</pad>标记是如何处理的，请作者们看一些这个问题。
